api/tests/unit/infrastructure/utils/string-utils_test.js

```diff
@@ -72,6 +72,10 @@ describe('Unit | Utils | string-utils', () => {
 
   describe('#normalizeAndSortChars', () => {
 
+    it('test', () => {
```

Comment:
should normalize and sort chars of the string "12345"

Reply:
Oui, j'ai oublié de changer le label du test après avoir fait l'implémentation 🤦.